src/components/SEP/Proposals/SEPProposalsAndAssignmentsTable.tsx

```diff
@@ -138,22 +149,86 @@ const SEPProposalsAndAssignmentsTable: React.FC<
     UserRole.USER_OFFICER,
   ]);
 
-  const removeProposalFromSEP = async (
-    proposalToRemove: SEPProposalType
+  /**
+   * NOTE: Custom action buttons are here because when we have them inside actions on the material-table
+   * and selection flag is true they are not working properly.
+   */
+  const RowActionButtons = (rowData: SEPProposalType) => {
+    const iconButtonStyle = { padding: '7px' };
+
+    return (
+      <>
+        <Tooltip title="View proposal">
+          <IconButton
+            data-cy="view-proposal"
+            onClick={() => {
+              setUrlQueryParams({ reviewModal: rowData.proposalPk });
+            }}
+            style={iconButtonStyle}
+          >
+            <Visibility />
+          </IconButton>
+        </Tooltip>
+        <Tooltip title="Assign SEP Member">
+          <IconButton
+            data-cy="assign-sep-member"
+            onClick={() => setProposalPk(rowData.proposalPk)}
+            style={iconButtonStyle}
+          >
+            <AssignmentInd />
+          </IconButton>
+        </Tooltip>
+      </>
+    );
+  };
+
+  const handleBulkDownloadClick = (
+    event: React.MouseEventHandler<HTMLButtonElement>,
+    rowData: SEPProposalType | SEPProposalType[]
+  ) => {
+    if (!Array.isArray(rowData)) {
+      return;
+    }
+
+    downloadPDFProposal(
```

Comment:
I think it could be confusing if you download more than one proposal but the filename is the first proposal's name. Could we do the same as on GoogleDrive where if only one proposal is downloaded then the filename is the proposal title, but if more than one, then have a generic name. For GoogleDrive it is f.x. `drive-download-20220420T092007Z-001.zip`. We could have `uos-proposals-current-date`

Reply:
@jekabs-karklins I think we do that for the file name already because it is generated on the backend. But this is only what is shown in the box while downloading in the frontend. I see your point and it is a good one. I will just change that to be same as user officer proposals table where it says: 2 proposals selected
![Screenshot_33](https://user-images.githubusercontent.com/6333063/164222032-23924dc1-b095-4f14-9175-226082eb57a6.png)